2016/webcam_record/webcam_record.cbp

```diff
@@ -14,6 +14,14 @@
 				<Compiler>
 					<Add option="-g" />
 				</Compiler>
+				<Linker>
+					<Add library="/home/benjamin/opencv-3.1.0/build/lib/libopencv_core.so" />
+					<Add library="/home/benjamin/opencv-3.1.0/build/lib/libopencv_highgui.so" />
+					<Add library="/home/benjamin/opencv-3.1.0/build/lib/libopencv_imgcodecs.so" />
+					<Add library="/home/benjamin/opencv-3.1.0/build/lib/libopencv_imgproc.so" />
+					<Add library="/home/benjamin/opencv-3.1.0/build/lib/libopencv_videoio.so" />
+					<Add library="/home/benjamin/2702_2016/2016/opencv_lib/bin/Debug/libopencv_lib.a" />
+				</Linker>
```

Comment:
Same deal.  We can't keep switching lib directories, so we'll define a OPENCV_LIB path, and then all these will be referenced from that.